e2e/tests/upgrades/upgrade_test.go

```diff
@@ -635,11 +626,8 @@ func (s *UpgradeTestSuite) TestV7ChainUpgradeAddLocalhost() {
 }
 
 // RegisterInterchainAccount will attempt to register an interchain account on the counterparty chain.
-func (s *UpgradeTestSuite) RegisterInterchainAccount(ctx context.Context, chain *cosmos.CosmosChain, user ibc.Wallet, msgRegisterAccount *intertxtypes.MsgRegisterAccount) error {
-	txResp, err := s.BroadcastMessages(ctx, chain, user, msgRegisterAccount)
-	s.Require().NoError(err)
-	s.AssertValidTxResponse(txResp)
-	return err
+func (s *UpgradeTestSuite) RegisterInterchainAccount(ctx context.Context, chain *cosmos.CosmosChain, user ibc.Wallet, msgRegisterAccount *intertxtypes.MsgRegisterAccount) sdk.TxResponse {
```

Comment:
I added this back in since it seemed like an odd change. The functions that currently wrap message creation currently sometimes create the message themselves (when the type of the message is consistent) or they take it as an arg when it isn't (like `RegisterInterchainAccount`).

Cases like the second might need some refactoring out since they have a very dubious existence.